### DISCUSSION TITLE
Download Archives

### DISCUSSION THREAD

**Discussion by tattootoe99** [Security-Onion-Solutions/securityonion#9898] [General] [answered]
Hello, my name is Doug. I am a Systems Engineer for a State College and we have an environment that uses Security Onion for students to learn. We moved the environment to new storage but failed to move the 16.04.6.6 iso. Do you know where I can download a replacement?

TIA,

Doug

**Comment by reyesj2** [comment, marked as answer]
I would strongly recommend using the latest Security Onion ISO. This brings the students up to date with the latest developments from Security Onion Solutions and allows them to better leverage the currently SO community for any questions.

**Reply by tattootoe99** [reply]
Hello Reyesj2,

Unfortunately I cannot use the latest until the curriculum is rewritten with labs for the latest distro. I need to get through this stretch and then we can rewrite using v2.

Doug

**Reply by reyesj2** [reply]
With 16.04 having reached end of life in April of 2021 we no longer provide support or downloads.

Security Onion 16.04 reached End Of Life on April 16, 2021:
https://docs.securityonion.net/en/2.3/eol.html
https://blog.securityonion.net/2021/04/security-onion-1604-has-reached-end-of.html

Check out our free training:
https://www.youtube.com/@security-onion/videos
